### DISCUSSION TITLE
back-to-basics-azdo-backlog-automation

### DISCUSSION THREAD

**Discussion by giscus** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#26] [Technical Blog]
# back-to-basics-azdo-backlog-automation

https://wsbctechnicalblog.github.io/back-to-basics-azdo-backlog-automation.html

**Comment by PamC-2024** [comment]
Great article Willy! And thank you for the note about portfolio and scorecard backlogs, that's helpful to know ahead.
